druid/examples/multiwin.rs

```diff
@@ -79,7 +81,58 @@ fn ui_builder() -> impl Widget<State> {
     row.add_child(Padding::new(5.0, inc_button));
     row.add_child(Padding::new(5.0, dec_button));
     col.add_flex_child(Align::centered(row), 1.0);
-    col
+    Glow::new(col)
+}
+
+struct Glow<W> {
+    inner: WidgetPod<State, W>,
+}
+
+impl<W: Widget<State>> Glow<W> {
```

Comment:
just a note: I think you could have achieved this using a `Painter`?

Reply:
Perhaps in some way, but it wasn't super obvious. Like the `Painter` docs state: *If you would like it to repaint at other times (such as when hot or active state changes) you will need to call `request_paint` further up the tree.* There's nothing further up the tree, so it wouldn't just be a `Painter` at least.

Reply:
oh huh for some reason I thought it did repaint on hot. My mistake!

I wonder if it makes sense to just make 'paint on hot' be an option for painter? but that's a small thing.